FaunaDB.Client/Client/FaunaClient.cs

```diff
@@ -245,10 +248,15 @@ public static ObjectV FromJson(string json)
             string endpoint,
             TimeSpan? timeout = null,
             HttpClient httpClient = null,
-            Version httpVersion = null)
+            Version httpVersion = null,
+            bool checkNewVersion = true)
         {
             secret.AssertNotNull(nameof(secret));
             endpoint.AssertNotNull(nameof(endpoint));
+            if (!checkNewVersion)
```

Comment:
This `if` doesn't look good. Can we just pass a `checkNewVersion` param into `DefaultClientIO` and add it into conditional there:
```
if (checkNewVersion && !Utils.CheckLatestVersion.AlreadyChecked.HasValue) 
```